integrasjon/oppgave-rest-klient/src/main/java/no/nav/vedtak/felles/integrasjon/oppgave/v1/Oppgave.java

```diff
@@ -19,6 +20,8 @@ public record Oppgave(Long id,
                       LocalDate fristFerdigstillelse,
                       LocalDate aktivDato,
                       Prioritet prioritet,
-                      Oppgavestatus status) {
-
+                      Oppgavestatus status,
+                      String orgnr,
```

Comment:
Trenger vi orgnr? Fpfordel oppretter kun oppgaver for bruker.

Reply:
Hva med inntektsmeldinger?

Reply:
De opprettes på bruker. Fpfordel henter ut bruker fra innholdet og setter bruker på journalpost + oppgave. AvsenderMottaker er satt til bedrit